client/src/components/page/Home.js

```diff
@@ -32,6 +33,14 @@ class Home extends Component {
           afraid to get creative! You could start by modifying this page to tell
           the world more about your team.
         </p>
+        <p>
+          Below you will find a map of Googleplex in Mountain View, California.
+        </p>
+        <GoogleMapReact
+          bootstrapURLKeys={{ key: 'AIzaSyAi9TMtkY74gzfmjPkD7w1Tu-zyABHYlww' }}
+          defaultCenter={{ lat: 37.422, lng: -122.084 }}
```

Comment:
Let's make these magic numbers constant variables at the top under the imports.

cosnt GOOGLE_MAPS_APP_KEY = 'AIzaSyAi9TMtkY74gzfmjPkD7w1Tu-zyABHYlww'
const GOOGLEPLEX_COORD = { lat:37.422, lng: -122.084 };
const DEFAULT_MAP_ZOOM = 16;